net/seafile-seahub/Makefile

```diff
@@ -8,16 +8,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=seafile-seahub
-PKG_VERSION:=6.3.0
-PKG_RELEASE=$(PKG_SOURCE_VERSION)-1
+PKG_VERSION:=6.3.7
+PKG_RELEASE:=1
 PKG_LICENSE:=Apache-2.0
 
-PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://github.com/haiwen/seahub.git
-PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_SOURCE_VERSION:=d1ab146a936a6ea1e1581bf3f194e86742f0d3cd
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
-PKG_MIRROR_HASH:=21c2ed3886d0c87853f9720704b14c0a0dce8215e7346af775e306f80b29fa0d
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/haiwen/seahub/tar.gz/v$(PKG_VERSION)-pro?
```

Comment:
Be careful, you are switching the codebase to the Professional Edition here which is only free for up to 3 users. Also, the [EULA](https://download.seafile.com/d/6e5297246c/files/?p=/pro/seafile_professional_edition_software_license_agreement.md) has a "No Derivative Works" clause which I believe prevents the software to be bundled as a package in other projects (I'm not 100% sure about that, though).

I'd advise to stick with Community Edition to avoid possibly breaking the EULA or making users accidentally install Seahub without purchasing a proper licence first.

Reply:
Didn't realize there was a distinction. Fixed.